modules/help.sh

```diff
@@ -108,6 +108,7 @@ convert <input file> <output file>        converts video and audio files
 gpio                                      displays raspberry pi model and corresponding GPIO ports
 changelog                                 displays the most recent changes to treehouses
 shutdown [now|in|force]                   shutdown the system
+magazine magpi [all|latest|number]        downloads specific magpi issue as a pdf
```

Comment:
I remember @dogi suggested me to leave `shutdown` to be the very last, not a serious issue though.